src/brouwers/kitreviews/models.py

```diff
@@ -95,7 +91,7 @@ class KitReviewVote(models.Model):
     VOTE_TYPES = (
```

Comment:
So this will be smth like:
`Class VoteType(DjangoChoices):
    plus = Choiceitem('+', label='+') # I can't use '+' or '-' as prop name
    ....`